test/e2e/query_test.go

```diff
@@ -166,7 +166,9 @@ func instantQuery(t *testing.T, ctx context.Context, addr string, q string, opts
 
 	fmt.Println("queryAndAssert: Waiting for", expectedSeriesLen, "results for query", q)
 	var result model.Vector
-	testutil.Ok(t, runutil.RetryWithLog(log.NewLogfmtLogger(os.Stdout), time.Second, ctx.Done(), func() error {
+	logger := log.NewLogfmtLogger(os.Stdout)
+	logger = log.With(logger, "ts", log.DefaultTimestampUTC)
```

Comment:
hmmm it's fancy, but wonder if there is some native way to find log ts... anyway. LGTM for now.